pkg/asset/cluster/cluster.go

```diff
@@ -74,6 +75,10 @@ func (c *Cluster) Generate(parents asset.Parents) (err error) {
 	terraformVariables := &TerraformVariables{}
 	parents.Get(clusterID, installConfig, terraformVariables)
 
+	if fs := installConfig.Config.FeatureSet; fs != configv1.Default {
+		logrus.Warnf("FeatureSet %q is enabled. This may affect the supportability of the cluster.", fs)
```

Comment:
nit: are there any feature sets that do not affect supportability?  I realize you say `may` here, but it's also `warn`.  One generic approach would be `Infof` without the supportability sentence.  Another approach would be adding the supportability caveat only for an allow-list of support-impacting feature sets.  Probably other alternatives too.  But if there are supported feature sets, we don't want to spook users who enable them.